script.js

```diff
@@ -28,3 +50,179 @@ window.addEventListener('resize', () => {
+</button>
+
+<button class="P-button">
+    ${mobilePopUp.button_name2}
+    <img src="${mobilePopUp.button_image2}" alt="Github icon">
+</button>
+</div>
+</div>
+
+<ul class="flex">
+<li class="flex tags">
+  ${desktopPopUp.tags[0]}
+</li>
+
+<li class="flex tags">
+  ${desktopPopUp.tags[1]}
+</li>
+
+<li class="flex tags">
+  ${desktopPopUp.tags[2]}
+</li>
+
+<li class="flex tags">
+  ${desktopPopUp.tags[3]}
+</li>
+
+<li class="flex tags">
+  ${desktopPopUp.tags[4]}
+</li>
+
+<li class="flex tags">
+  ${desktopPopUp.tags[5]}
+</li>
+</ul>
+
+<p>
+${mobilePopUp.project_description}
+</p>
+</div>
+</div>`;
+
+pB2.innerHTML = desktopdesktopPopUp;
+
+// Project section of the page.
+const ProjectCardContent = {
+  title1: 'Multi-Post Stories',
+  title2: 'Gain+Glory',
+  project_description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+  tags: ['Ruby on rails', 'css', 'javascript', 'html'],
+  button_name: 'See Project',
+};
+
+const projects = `<div class="Card">
+
+<section class="flex card-content">
+    <div>
+        <h3>
+           ${ProjectCardContent.title1} 
+        </h3>
+        <h3>
+        ${ProjectCardContent.title2}
+        </h3>
+    </div>
+
+    <ul class="flex">
+        <li class="flex tags">
+            ${ProjectCardContent.tags[0]}
+        </li>
+
+        <li class="flex tags">
+        ${ProjectCardContent.tags[1]}
+        </li>
+
+        <li class="flex tags">
+        ${ProjectCardContent.tags[2]}
+        </li>
+
+        <li class="flex tags">
+        ${ProjectCardContent.tags[3]}
+        </li>
+    </ul>
+
+    <button class="flex btn" id="btn-display">
+        ${ProjectCardContent.button_name}
+    </button>
+</section>
+
+</div>
+`;
+
+// eslint-disable-next-line no-plusplus
+for (let i = 0; i < 6; i++) {
+  projectCards.innerHTML += projects;
+}
+
+const buttonDisplay = Array.from(document.querySelectorAll('#btn-display'));
+buttonDisplay.forEach((button) => {
+  button.addEventListener('click', () => {
+    if (body.clientWidth >= 992) {
+      document.getElementById('modal-desktop').style = 'display: flex; z-index: 3';
+    } else if (body.clientWidth < 992) {
+      document.getElementById('modal').style = 'display: flex';
+    }
+  });
+});
+
+document.getElementById('btn-hide').addEventListener('click', () => {
+  if (body.clientWidth >= 992) {
+    document.getElementById('modal-desktop').style = 'display: none';
+  } else if (body.clientWidth < 992) {
+    document.getElementById('modal').style = 'display: none';
```

Comment:
- [ ] <b>REQUIRED:</b> The same as here, you don't need to check whether you are on desktop or mobile just to use a different modal, you can use one and let CSS take care of the rest.